discord/user.go

```diff
@@ -6,25 +6,28 @@ import (
 )
 
 type User struct {
-	ID            UserID `json:"id"`
+	// may be ommited
+	Email         string `json:"email,omitempty"`
```

Comment:
Remove this comment, since it violates the godoc format, is inconsitent with other structs that don't offer that information in their docs, and the `omitempty` option of the `json` tag specifies omittability already.

```suggestion
	Email         string `json:"email,omitempty"`
```